.gitignore

```diff
@@ -8,4 +8,4 @@ target/
 .*.swp
 *~
 *.bak
-
+\.env
```

Comment:
Is `.env` a directory in the working directory? Is there no platform-independent solution?